README.md

```diff
@@ -105,36 +105,27 @@ DOCKER IMAGES
 Docker images are build by travis-ci and pushed to [hub.docker.com](https://hub.docker.com/r/ergw/ergw-c-node/tags),
 and by gitlab.com and pushed to [quay.io](https://quay.io/repository/travelping/ergw-c-node?tab=tags).
 
-BUILDING
---------
-
-*The minimum supported Erlang version is 22.3.4.*
-
-Erlang 23.0.3 is the recommended version.
-
-Using rebar:
-
-    # rebar3 compile
-
-RUNNING
+BUILDING & RUNNING
```

Comment:
Good explanation that could perhaps go into a separate commit.

Reply:
Moved README changes into https://github.com/travelping/ergw/pull/214